### ISSUE TITLE
Byt namn på projektet!

### ISSUE THREAD

**Issue by johkin** [johkin/ihome#2]
namnet ihome verkar vara associerat med högtalare för iphones mm, så man kanske borde tänka ut något fiffigare som man inte får så många google-träffar på?
Något med "open"?
OpenHome verkar också vara taget.

Det vore coolt med en *.io-adress?  :smiley_cat:

**Comment by frojdh** [comment]
Ja, jag såg också det. Kändes lite apple också då dom har i som prefix på en del.
Några simpla förslag:
- ctrlHome: känns lite instängt...
- OcH, Open Connected Home 

*io-adress köper jag
